<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regression</class>
 <widget class="QGroupBox" name="regression">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>599</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>999999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Regression</string>
  </property>
  <property name="title">
   <string>Regression - Train</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="regression_vlayout">
     <item>
      <layout class="QHBoxLayout" name="regression_choosedata_hlayout">
       <item>
        <widget class="QLabel" name="regression_choosedata_label">
         <property name="text">
          <string>Choose data: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="regression_choosedata">
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Choose Data</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Known Data</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="regression_choosedata_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="regression_choosevars_hlayout">
       <item>
        <layout class="QVBoxLayout" name="regression_choosex_vlayout">
         <item>
          <widget class="QLabel" name="regression_choosex_label">
           <property name="text">
            <string>Choose X variable(s): </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="regression_choosex">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Choose X</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="regression_choosey_vlayout">
         <item>
          <widget class="QLabel" name="regression_choosey_label">
           <property name="text">
            <string>Choose Y variable(s):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="regression_choosey">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Choose Y</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="yrange_vlayout">
         <item>
          <layout class="QHBoxLayout" name="ymin_hlayout">
           <item>
            <widget class="QLabel" name="ymin_label">
             <property name="text">
              <string>Y min:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="ymin_spin"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="ymax_hlayout">
           <item>
            <widget class="QLabel" name="ymax_label">
             <property name="text">
              <string>Y max:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="ymax_spin"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ransac_hlayout">
       <item>
        <widget class="QCheckBox" name="regression_ransac_checkbox">
         <property name="text">
          <string>RANSAC</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ransac_options_hlayout">
         <item>
          <widget class="QLabel" name="ransac_lossfunc_label">
           <property name="text">
            <string>Loss function:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ransac_lossfunc_combobox">
           <item>
            <property name="text">
             <string>Squared</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Absolute</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ransac_thresh_label">
           <property name="text">
            <string>Threshold:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="ransac_thresh_spin"/>
         </item>
         <item>
          <spacer name="ransac_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="regression_choosealg_hlayout">
       <item>
        <widget class="QLabel" name="regression_choosealg_label">
         <property name="text">
          <string>Choose Algorithm: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="regression_choosealg">
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>PLS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Others coming soon...</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="regression_choosealg_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="pls_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="pls_nc_label">
          <property name="text">
           <string># of Components: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pls_nc_spinbox"/>
        </item>
        <item>
         <spacer name="pls_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="gp_widget" native="true">
       <layout class="QVBoxLayout" name="gp_vlayout">
        <item>
         <layout class="QHBoxLayout" name="gp_dim_red_hlayout">
          <item>
           <widget class="QLabel" name="gp_dim_red_label">
            <property name="text">
             <string>Dimensionality reduction method:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="gp_dim_red_combobox">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>PCA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FastICA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ICA - JADE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="gp_dim_red_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="gp_verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="gp_rand_starts_hlayout">
            <item>
             <widget class="QLabel" name="gp_rand_starts_label">
              <property name="text">
               <string># of random starts:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="gp_rand_starts_spin">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="gp_rand_starts_spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="gp_theta0_hlayout">
            <item>
             <widget class="QLabel" name="gp_theta0_label">
              <property name="text">
               <string>Starting Theta:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="gp_theta0_spin">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="gp_theta0_spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="gp_thetaU_hlayout">
            <item>
             <widget class="QLabel" name="gp_thetaU_label">
              <property name="text">
               <string>Upper bound on Theta:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="gp_thetaU_spin">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="gp_thetaU_spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="gp_thetaL_hlayout">
            <item>
             <widget class="QLabel" name="gp_thetaL_label">
              <property name="text">
               <string>Lower bound on Theta:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="gp_thetaL_spin">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="gp_thetaL_spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
